### DISCUSSION TITLE
Will there be an update for 24H2?

### DISCUSSION THREAD

**Discussion by artad-sys** [valinet/ExplorerPatcher#3978] [General]
Currently if I install EP, when I launch it, it says in the options: Windows 10 taskbar is not available. I use Windows 11 Pro and it's not a company machine.

**Comment by pyrates999** [comment]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

**Comment by artad-sys** [comment]
D*mn...I'm such an *ssh0le....I didn't click on "Restart File Explorer"' after I changed it to "Windows 10 (ExplorerPatcher) so I thought it doesn't work and some updates will fix it...lol....and I was waiting for an update for 2 or 3 weeks now...
Thanks man, I love you all... :)

**Reply by Amrsatrio** [reply]
Bruh
